{{cookiecutter.project_slug}}/.gitignore

```diff
@@ -323,13 +323,7 @@ Session.vim
 tags
 {% if cookiecutter.use_docker == 'n' %}
 
-### VirtualEnv template
 # Virtualenv
-[Bb]in
-[Ii]nclude
-[Ll]ib
-[Ll]ib64
-[Ss]cripts
```

Comment:
I had to remove the `[Bb]in` entry otherwise it was ignored, and I ended up removing all these, but now I stumbled upon this one #2226 

I'm not a Windows user so I'm not sure how these work. Need to look into this.

Reply:
I've changed this to be more conservative, and only remove the line we need for this PR: `[Bb]in`.